pallets/confidential/src/lib.rs

```diff
@@ -70,6 +75,10 @@ decl_storage! {
         pub RangeProofs get(fn range_proof): double_map hasher(twox_64_concat) IdentityId, hasher(blake2_128_concat) ProverTickerKey => Option<TickerRangeProof>;
 
         pub RangeProofVerifications get(fn range_proof_verification): double_map hasher(blake2_128_concat) (IdentityId, Ticker), hasher(twox_64_concat) IdentityId => bool;
+
+        pub MercatAccounts get(fn mercat_accounts): map hasher(blake2_128_concat) IdentityId => Vec<PubAccount>;
```

Comment:
Can we hold something like that 
```
pub MercatAccounts get(fn mercat_accounts): double_map hasher(blake2_128_concat) asset_id, hasher(blake2_128_concat) IdentityId => PubAccount;
```
It will allow accessing the list of all identity who has the given asset id easily and also it will save us from encoding/decoding of a vector on-chain when we need to add/delete the account or access the account for a given asset_id